lib/testing/test_transaction.ex

```diff
@@ -1,67 +1,65 @@
-if Mix.env == :test do
-  defmodule Braintree.Testing.TestTransaction do
-    @moduledoc """
-    Create transasctions for testing purposes only.
-    Transition to settled, settlement_confirmed, or settlement_declined states.
-    """
+defmodule Braintree.Testing.TestTransaction do
+  @moduledoc """
+  Create transasctions for testing purposes only.
```

Comment:
This should have had a blank line after the first sentence, could you add that since you're touching it?

Reply:
And a typo in transactions fixed as well.